src/js/components/Collapsible/__tests__/Collapsible-test.js

```diff
@@ -0,0 +1,150 @@
+      <Grommet>
+        <Button
+          onClick={() => {
+            open = !open;
+            useEffect();
+          }}
+          label="Toggle"
+        />
+        <Collapsible id="hidden-test" open={open}>
+          <Box>
+            <Text>Example</Text>
+          </Box>
+        </Collapsible>
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.click(getByText('Toggle'));
+    expect(open).toBe(true);
+    rerender(
+      <Grommet>
+        <Button
+          onClick={() => {
+            open = !open;
+            useEffect();
+          }}
+          label="Toggle"
+        />
+        <Collapsible id="hidden-test" open={open}>
+          <Box>
+            <Text>Example</Text>
+          </Box>
+        </Collapsible>
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+    expect(useEffect).toBeCalledTimes(1);
+  });
+
+  test('rerender-direction-vertical', () => {
+    const useEffect = jest.fn();
+    let open = false;
+    const { container, rerender, getByText } = render(
+      <Grommet>
+        <Button
+          onClick={() => {
+            open = !open;
+            useEffect();
+          }}
+          label="Toggle"
+        />
+        <Collapsible id="hidden-test" open={open} direction="vertical">
+          <Box>
+            <Text>Example</Text>
+          </Box>
+        </Collapsible>
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.click(getByText('Toggle'));
+    expect(open).toBe(true);
+    rerender(
+      <Grommet>
+        <Button
+          onClick={() => {
+            open = !open;
+            useEffect();
+          }}
+          label="Toggle"
+        />
+        <Collapsible id="hidden-test" open={open} direction="vertical">
+          <Box>
+            <Text>Example</Text>
+          </Box>
+        </Collapsible>
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+    expect(useEffect).toBeCalledTimes(1);
+  });
+
+  test('rerender-direction-horizontal', () => {
+    const useEffect = jest.fn();
+    let open = false;
+    const { container, rerender, getByText } = render(
+      <Grommet>
+        <Button
+          onClick={() => {
+            open = !open;
+            useEffect();
+          }}
+          label="Toggle"
+        />
+        <Collapsible id="hidden-test" open={open} direction="horizontal">
+          <Box>
+            <Text>Example</Text>
+          </Box>
+        </Collapsible>
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.click(getByText('Toggle'));
+    expect(open).toBe(true);
+    rerender(
+      <Grommet>
+        <Button
+          onClick={() => {
+            open = !open;
+            useEffect();
+          }}
+          label="Toggle"
+        />
+        <Collapsible id="hidden-test" open={open} direction="horizontal">
```

Comment:
```suggestion
        <Collapsible open={open} direction="horizontal">
```
Could be removed all over the file.